docs-ref-conceptual/create-an-azure-service-principal-azure-cli.md

```diff
@@ -16,110 +16,59 @@ ms.assetid: fab89cb8-dac1-4e21-9d34-5eadd5213c05
 
 # Create an Azure service principal with Azure CLI 2.0
 
-If you plan to manage your app or service with Azure CLI 2.0,
-you should run it under an Azure Active Directory (AAD) service principal rather than your own credentials.
-This topic steps you through creating a security principal with Azure CLI 2.0.
+If you want to manage permissions for your app or service, you can do so through creating a service principal under Azure Active Directory. A service principal will allow you to set per-app permissions restrictions. This topic will run you through the steps for creating a service principal and setting its permissions.
 
 > [!NOTE]
-> You can also create a service principal through the Azure portal.
-> Read [Use portal to create Active Directory application and service principal that can access resources](/azure/azure-resource-manager/resource-group-create-service-principal-portal) for more details.
-
-## What is a 'service principal'?
-
-An Azure service principal is a security identity used by user-created apps, services, and automation tools to access specific Azure resources. Think of it as a 'user identity' (login and password or certificate) with a specific role, and tightly controlled permissions to access your resources. It only needs to be able to do specific things, unlike a general user identity. It improves security if you only grant it the minimum permissions level needed to perform its management tasks. 
-
-Right now, Azure CLI 2.0 only supports the creation of password-based authentication credentials. In this topic, we cover creating a service principal with a specific password, and optionally assigning specific roles to it.
+> Right now, Azure CLI 2.0 only supports the creation of password-based authentication credentials. If you need to create certificate-based credentials, create the service principal through the Azure portal. Read [Use portal to create Active Directory application and service principal that can access resources](/azure/azure-resource-manager/resource-group-create-service-principal-portal) for more details.
```

Comment:
Might want to double check this, I believe cert-based options were added recently to the CLI options for az ad sp create